### PR TITLE
Fix: Setting #move_to should mark an orderable object as changed so that callbacks are triggered.

### PR DESCRIPTION
**Pull request by johnnyshields**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
-### 6.0.4 (Next)
+### 6.0.5 (Next)
 
 * Your contribution here.
 
+### 6.0.4 (Next)
```
Comment (dblock) [inline CHANGELOG.md:5]: I think you meant to leave this at 6.0.4.

File: lib/mongoid/orderable/mixins/changeable.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Mongoid
+module Orderable
```
Comment (dblock) [inline lib/mongoid/orderable/mixins/changeable.rb:4]: File needs some indent.

File: spec/integration/changeable_spec.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe Mongoid::Orderable::Mixins::Changeable do
+
```
Comment (dblock) [inline spec/integration/changeable_spec.rb:4]: Rubocop is missing ;)

File: lib/mongoid/orderable/mixins/changeable.rb (hunk 2 above)
Comment (johnnyshields) [reply lib/mongoid/orderable/mixins/changeable.rb:4]: In this project we aren't indenting multiple modules. Rubocop is set accordingly:

```
Layout/IndentationWidth:
  AllowedPatterns:
    - '^\s*module'
```

File: CHANGELOG.md (hunk 1 above)
Comment (johnnyshields) [reply CHANGELOG.md:5]: Fixed in a follow-up PR

File: spec/integration/changeable_spec.rb (hunk 3 above)
Comment (johnnyshields) [reply spec/integration/changeable_spec.rb:4]: Rubocop added here: https://github.com/mongoid/mongoid_orderable/pull/76

**Pull request merged**